docs/BreakingChanges/043 HttpRequest.ContentEncoding property prohibits UTF7.md

```diff
@@ -0,0 +1,24 @@
+## 43: HttpRequest.ContentEncoding property
```

Comment:
{
  "Id": "43",
  "Title": "HttpRequest.ContentEncoding property",
  "Details": "Beginning in .NET Framework 4.5, UTF-7 encoding is prohibited in HttpRequests' bodies. Data for applications that depend on incoming UTF-7 data will not decode properly in some cases.",
  "Suggestion": "Ideally, applications should be updated to not use UTF-7 encoding in HttpRequests. Alternatively, legacy behavior can be restored by using the `aspnet:AllowUtf7RequestContentEncoding` attribute of the [appSettings](https://msdn.microsoft.com/en-us/library/hh975440(v=vs.110).aspx) element.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#asp",
  "Markdown": null,
  "ApplicableApis": [
    "P:System.Web.HttpRequest.ContentEncoding",
    "M:System.Web.HttpRequest.get_ContentEncoding",
    "M:System.Web.HttpRequest.set_ContentEncoding(System.Text.Encoding)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": true,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] P:System.Web.HttpRequest.ContentEncoding
- [x] M:System.Web.HttpRequest.get_ContentEncoding
- [x] M:System.Web.HttpRequest.set_ContentEncoding(System.Text.Encoding)